modules/hcp-dc-mgmt-cluster.adoc

```diff
@@ -6,143 +6,28 @@
-Providing information about network ipv6...
-cidr: 2620:52:0:1306::/64
-dhcp: false
-domain: dns.base.domain.name
-mode: nat
-plan: kvirt
-type: routed
-----
+* The management cluster must be able to resolve and reach the API and `{asterisk}.apps` hostname of the hosted cluster. Here is an example of the API hostname of the hosted cluster and `{asterisk}.apps` hostname:
 
-. Ensure that the pull secret and `kcli` plan files are in place so that you can deploy the {product-title} management cluster:
+** `api.sno-hosted-cluster-1.internal.domain.com`
+** `console-openshift-console.apps.sno-hosted-cluster-1.internal.domain.com`
 
-.. Confirm that the pull secret is in the same folder as the `kcli` plan, and that the pull secret file is named `openshift_pull.json`.
+.Procedure
 
-.. Add the `kcli` plan, which contains the {product-title} definition, in the `mgmt-compact-hub-dual.yaml` file. Ensure that you update the file contents to match your environment:
-+
-[source,yaml]
-----
-plan: hub-dual
-force: true
-version: stable
-tag: "<4.x.y>-x86_64" <1>
-cluster: "hub-dual"
-dualstack: true
-domain: dns.base.domain.name
-api_ip: 192.168.126.10
-ingress_ip: 192.168.126.11
-service_networks:
-- 172.30.0.0/16
-- fd02::/112
-cluster_networks:
-- 10.132.0.0/14
-- fd01::/48
-disconnected_url: registry.dns.base.domain.name:5000
-disconnected_update: true
-disconnected_user: dummy
-disconnected_password: dummy
-disconnected_operators_version: v4.14
-disconnected_operators:
-- name: metallb-operator
-- name: lvms-operator
-  channels:
-  - name: stable-4.14
-disconnected_extra_images:
-- quay.io/user-name/trbsht:latest
-- quay.io/user-name/hypershift:BMSelfManage-v4.14-rc-v3
-- registry.redhat.io/openshift4/ose-kube-rbac-proxy:v4.10
-dualstack: true
-disk_size: 200
-extra_disks: [200]
-memory: 48000
-numcpus: 16
-ctlplanes: 3
-workers: 0
-manifests: extra-manifests
-metal3: true
-network: dual
-users_dev: developer
-users_devpassword: developer
-users_admin: admin
-users_adminpassword: admin
-metallb_pool: dual-virtual-network
-metallb_ranges:
-- 192.168.126.150-192.168.126.190
-metallb_autoassign: true
-apps:
-- users
-- lvms-operator
-- metallb-operator
-vmrules:
-- hub-bootstrap:
-    nets:
-    - name: ipv6
-      mac: aa:aa:aa:aa:10:07
-- hub-ctlplane-0:
-    nets:
-    - name: ipv6
-      mac: aa:aa:aa:aa:10:01
-- hub-ctlplane-1:
-    nets:
-    - name: ipv6
-      mac: aa:aa:aa:aa:10:02
-- hub-ctlplane-2:
-    nets:
-    - name: ipv6
-      mac: aa:aa:aa:aa:10:03
-----
-+
-<1> Replace `<4.x.y>` with the supported {product-title} version you want to use.
+. Install {mce-short} 2.2 or later on an {product-title} cluster. You can install {mce-short} as an Operator from the {product-title} OperatorHub. The HyperShift Operator is included with {mce-short}. For more information about installing {mce-short}, see "Installing and upgrading multicluster engine operator" in the Red{nbsp}Hat Advanced Cluster Management documentation.
```

Comment:
HyperShift for Agent provider was not supported in MCE-2.2 AFAIR. IIRC it started with MCE-2.4